src/rippled/lib/txSummary/NFTokenMint.js

```diff
@@ -1,4 +1,6 @@
-const { convertHexToString } = require('xrpl');
+function convertHexToString(hex, encoding = 'utf8') {
+  return Buffer.from(hex, 'hex').toString(encoding);
+}
```

Comment:
put this in a utils folder please - also put a comment indicating that it will not be required once we do `xrpl` correctly